File: pset1 - C/mario (less)/mario.c
#include <stdio.h>
#include <cs50.h>

//Main function without parameters
int main(void)
{
    //Variable
    int height = 0;

    //Loop to get the correct value
    while (height <= 1 || height > 9)
    {
        height = get_int("Height: ") + 1;
    }


    //logic to make the pyramid
    for (int i = 1; i < height; i++)
    {

        for (int j = 1; j < (height - i); j++)
        {
            printf(" ");
        }

        for (int j = (height - i); j < height; j++)
        {
            printf("#");
        }

        printf("\n");
    }

}

File: pset6 - python/mario (more)/mario.py
height = 0

# Loop to get the correct value
while (1 >= height or height > 9):
    try:
        height = int(input("Height: ")) + 1
    except:
        height = 0

# Logic to make the pyramid
for i in range(1, height):
    for j in range(1, height - i):
        print(" ",  end='')

    for j in range(height - i, height):
        print("#",  end='')

    print("  ",  end='')

    for j in range(height - i, height):
        print("#",  end='')

    print("")

